src/opentelemetry-core.module.ts

```diff
@@ -62,6 +61,7 @@ export class OpenTelemetryCoreModule implements OnApplicationShutdown, OnApplica
         TraceService,
         MetricService,
       ],
+      global: options.isGlobal,
```

Comment:
Hi there! Could you share some insights on why would you want this module to be globally optional?

Reply:
I only see the value on that if the default value is true. Although I'm not sure how you would use it with global = false.